package com.stefanini.hackathon.exception;

public class AlunoNotFoundException extends Exception{
    public AlunoNotFoundException(Long id) {
       super("Aluno n√£o encontrado com id:" +id);
    }
}
